packages/graphqlsp/src/fieldUsage.ts

```diff
@@ -123,7 +124,8 @@ const crawlScope = (
   originalWip: Array<string>,
   allFields: Array<string>,
   source: ts.SourceFile,
-  info: ts.server.PluginCreateInfo
+  info: ts.server.PluginCreateInfo,
+  inArrayMethod: boolean
```

Comment:
We need a bail when we are in one of our known array methods so we can still support cases like the following https://github.com/0no-co/GraphQLSP/blob/main/packages/example-tada/src/index.tsx#L47-L50